### PR TITLE
185454321 client service pane

### PR DESCRIPTION
**Pull request by ZachCMP**
https://www.pivotaltracker.com/story/show/185454321

Relies on backend PR: https://github.com/greenriver/hmis-warehouse/pull/3192

### CODE REVIEW

File: src/components/clientDashboard/enrollments/ClientServices.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import { Paper, Stack, Typography } from '@mui/material';
```
Comment (gigxz) [inline src/components/clientDashboard/enrollments/ClientServices.tsx]: Can we move this file to `src/modules/services/components/`?

**Comment by gigxz** [comment]
You'll need to update [this file](https://github.com/greenriver/hmis-frontend/blob/6ae90a80638c509a36e8f1c626b71e98f0c8261d/src/components/layout/dashboard/contextHeader/useDashboardBreadcrumbs.tsx#L173-L176) to get breadcrumbs showing up
Screen Shot 2023-06-26 at 11 16 01 AM

**Review by gigxz — COMMENTED** [review]
Looks good, the breadcrumbs just need to be fixed & moving file to services module. Thanks!

**Comment by ZachCMP** [comment]
@gigxz Fixed

**Review by gigxz — APPROVED** [review]
LGTM, waiting to merge until backend is ready

**Pull request merged**